spec/v2/index.bs

```diff
@@ -90,10 +90,10 @@ The license can be found in [[#license]].
     See [[!DATA-MODEL-EXTENSIONS]] and [[!EXTENSIONS-GUIDANCE]] for further details.
 
 : <dfn>Data recipient</dfn>
-:: The [=SCA|Supply Chain Actor=] requesting and/or receiving [=PCF=] data from another [=SCA=].
+:: The company requesting and/or receiving [=PCF=] data from another company, using the technical means specified in this document.
```

Comment:
Definitely not intentional to break those links; they should be removed. Don't think it is necessary to also include a definition for company, because the actually relevant terms are **data owner** and **date recipient** which are defined and used consequently throughout the specs.